conftest.py

```diff
@@ -1,9 +1,11 @@
 import asyncio
+
 import pytest
 from gremlin_python.process.traversal import Cardinality
+
 from goblin import Goblin, driver, element, properties
-from goblin.driver import (
-    Connection, DriverRemoteConnection, GraphSONMessageSerializer)
+from goblin.driver import Connection, DriverRemoteConnection, \
+    GraphSONMessageSerializer
```

Comment:
I don't understand this change. Is this PEP8?

Reply:
I just run a default `isort`. Do you have any preference on the style: https://github.com/timothycrosley/isort#multi-line-output-modes

Reply:
No, it's `isort` (see https://github.com/timothycrosley/isort#multi-line-output-modes)

Reply:
I think I prefer: 
4 - Hanging Grid